timesketch/frontend-ng/src/components/Explore/EventList.vue

```diff
@@ -514,6 +529,7 @@ export default {
         suspicious: { color: 'orange', textColor: 'white', label: 'mdi-help-circle-outline' },
       },
       searchInProgress: false,
+      fullscreenLoading: false,
```

Comment:
Why is it called `fullscreenLoading`? If I understand the code correct, it is used as v-model for the export dialog.
Wouldn't something like `exportDialog` make more sense?

Reply:
It used to be a fullscreen loading element and not a dialoge. Let me rename it.